### DISCUSSION TITLE
Adding an option to prevent new user registration

### DISCUSSION THREAD

**Discussion by trapassati** [dani-garcia/vaultwarden#1933] [Ideas]
It would be useful to add an option to prevent the registration of new users

**Comment by jjlin** [comment]
See https://github.com/dani-garcia/vaultwarden/wiki/Disable-registration-of-new-users

**Comment by meichthys** [comment]
I think you can do what you want via the Admin panel:
![image](https://user-images.githubusercontent.com/10717998/154567779-2f48593e-f6f8-4066-bc4a-f8c09f31e1ab.png)
